src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -1105,7 +1105,12 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
                         var refPeStreamProviderOpt = finalRefPeFilePath != null ? new CompilerEmitStreamProvider(this, finalRefPeFilePath) : null;
 
                         RSAParameters? privateKeyOpt = null;
-                        if (compilation.Options.StrongNameProvider != null && compilation.SignUsingBuilder && !compilation.Options.PublicSign)
+                        if (compilation is
+                        {
+                            Options: { StrongNameProvider: { }, PublicSign: false },
+                            SignUsingBuilder: true
+                        }
+)
```

Comment:
`)` on column 1?

Reply:
➡️ Treating this one as a duplicate of https://github.com/dotnet/roslyn/pull/39738#discussion_r344383849